crowbar_framework/app/views/sanities/show.html.haml

```diff
@@ -0,0 +1,4 @@
+.row
+  .col-lg-12
+    .alert.alert-danger
+      = flash_for_sanity_checks(@errors)
```

Comment:
> Can we show a green alert when there's no error (since the page is always reachable)?

actually the site is always reachable, but if there are no errors we will be redirected. nevertheless I will add a check